src/audio.c

```diff
@@ -173,6 +184,14 @@ struct aurx {
 	enum jbuf_type jbtype;       /**< Jitter buffer type               */
 	volatile int32_t wcnt;       /**< Write handler call count         */
 
+#ifdef HAVE_PTHREAD
+	struct {
+		thrd_t thrd;         /**< Audio RX filter thread           */
+		mtx_t mtx;           /**< Mutex for stopping thread        */
```

Comment:
is it possible to use Atomic bool here, instead of mutex ?

Reply:
I think so. This mtx solution is maybe overcautious. I saw this in module ausine.